### ISSUE TITLE
Tartube won't start

### ISSUE THREAD

**Issue by harrwester** [axcore/tartube#245]
I installed Tartube (v2.3.008 latest version at that time) as .deb (https://sourceforge.net/projects/tartube/files/) on my LM20 desktop:

```
mint@harwester:~$ inxi -Fxz
System:
  Kernel: 5.4.0-60-generic x86_64 bits: 64 compiler: gcc v: 9.3.0 
  Desktop: Cinnamon 4.6.7 Distro: Linux Mint 20 Ulyana 
  base: Ubuntu 20.04 focal 
Machine:
  Type: Desktop Mobo: ASUSTeK model: Z170 PRO GAMING v: Rev X.0x 
  serial: <filter> UEFI [Legacy]: American Megatrends v: 1204 
  date: 01/19/2016 
CPU:
  Topology: Quad Core model: Intel Core i7-6700 bits: 64 type: MT MCP 
  arch: Skylake-S rev: 3 L2 cache: 8192 KiB 
  flags: avx avx2 lm nx pae sse sse2 sse3 sse4_1 sse4_2 ssse3 vmx 
  bogomips: 54398 
  Speed: 800 MHz min/max: 800/4000 MHz Core speeds (MHz): 1: 800 2: 800 
  3: 800 4: 800 5: 800 6: 800 7: 802 8: 800 
Graphics:
  Device-1: NVIDIA GM206 [GeForce GTX 960] vendor: eVga.com. driver: nvidia 
  v: 460.32.03 bus ID: 01:00.0 
  Display: x11 server: X.Org 1.20.9 driver: nvidia 
  resolution: 1920x1080~60Hz, 1920x1080~60Hz 
  OpenGL: renderer: GeForce GTX 960/PCIe/SSE2 v: 4.6.0 NVIDIA 460.32.03 
  direct render: Yes 
Audio:
  Device-1: Intel 100 Series/C230 Series Family HD Audio vendor: ASUSTeK 
  driver: snd_hda_intel v: kernel bus ID: 00:1f.3 
  Device-2: NVIDIA GM206 High Definition Audio vendor: eVga.com. 
  driver: snd_hda_intel v: kernel bus ID: 01:00.1 
  Sound Server: ALSA v: k5.4.0-60-generic 
Network:
  Device-1: Intel Ethernet I219-V vendor: ASUSTeK driver: e1000e v: 3.2.6-k 
  port: f000 bus ID: 00:1f.6 
  IF: enp0s31f6 state: up speed: 100 Mbps duplex: full mac: <filter> 
  IF-ID-1: tun0 state: unknown speed: 10 Mbps duplex: full mac: N/A 
Drives:
  Local Storage: total: 1.13 TiB used: 210.29 GiB (18.2%) 
  ID-1: /dev/sda vendor: Kingston model: SV300S37A240G size: 223.57 GiB 
  ID-2: /dev/sdb vendor: Western Digital model: WD10EZRX-22A3KB0 
  size: 931.51 GiB 
Partition:
  ID-1: / size: 218.57 GiB used: 111.20 GiB (50.9%) fs: ext4 dev: /dev/sda5 
Sensors:
  System Temperatures: cpu: 29.8 C mobo: 27.8 C gpu: nvidia temp: 52 C 
  Fan Speeds (RPM): N/A gpu: nvidia fan: 0% 
Info:
  Processes: 322 Uptime: 2h 52m Memory: 31.28 GiB used: 2.83 GiB (9.1%) 
  Init: systemd runlevel: 5 Compilers: gcc: 9.3.0 Shell: bash v: 5.0.17 
  inxi: 3.0.38
```
After couple of minutes of checking it out it crashed and won't start ever since:

```
mint@harwester:~$ tartube
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/tartube/mainapp.py", line 2435, in do_activate
    self.start()
  File "/usr/lib/python3/dist-packages/tartube/mainapp.py", line 2600, in start
    new_config_flag = self.load_config()
  File "/usr/lib/python3/dist-packages/tartube/mainapp.py", line 3294, in load_config
    LOCALE = gettext.translation(
  File "/usr/lib/python3.8/gettext.py", line 603, in translation
    raise FileNotFoundError(ENOENT,
FileNotFoundError: [Errno 2] No translation file found for domain: 'base'
Error in sys.excepthook:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/apport_python_hook.py", line 153, in apport_excepthook
    with os.fdopen(os.open(pr_filename,
FileNotFoundError: [Errno 2] No such file or directory: '/var/crash/_usr_bin_tartube.1000.crash'

Original exception was:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/tartube/mainapp.py", line 2435, in do_activate
    self.start()
  File "/usr/lib/python3/dist-packages/tartube/mainapp.py", line 2600, in start
    new_config_flag = self.load_config()
  File "/usr/lib/python3/dist-packages/tartube/mainapp.py", line 3294, in load_config
    LOCALE = gettext.translation(
  File "/usr/lib/python3.8/gettext.py", line 603, in translation
    raise FileNotFoundError(ENOENT,
FileNotFoundError: [Errno 2] No translation file found for domain: 'base'
```
Tried to completely remove it and reinstall it with no result. After the latest version (v2.3.042) came out I reinstalled it again, but still won't start. I read somewhere it might be ffmpeg related and I even get 'WARNING: library configuration mismatch' when running ffmpeg in terminal, but not sure if this really is the problem.

```
mint@harwester:~$ ffmpeg
ffmpeg version 4.2.4-1ubuntu0.1 Copyright (c) 2000-2020 the FFmpeg developers
  built with gcc 9 (Ubuntu 9.3.0-10ubuntu2)
  configuration: --prefix=/usr --extra-version=1ubuntu0.1 --toolchain=hardened --libdir=/usr/lib/x86_64-linux-gnu --incdir=/usr/include/x86_64-linux-gnu --arch=amd64 --enable-gpl --disable-stripping --enable-avresample --disable-filter=resample --enable-avisynth --enable-gnutls --enable-ladspa --enable-libaom --enable-libass --enable-libbluray --enable-libbs2b --enable-libcaca --enable-libcdio --enable-libcodec2 --enable-libflite --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-libgme --enable-libgsm --enable-libjack --enable-libmp3lame --enable-libmysofa --enable-libopenjpeg --enable-libopenmpt --enable-libopus --enable-libpulse --enable-librsvg --enable-librubberband --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libssh --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx265 --enable-libxml2 --enable-libxvid --enable-libzmq --enable-libzvbi --enable-lv2 --enable-omx --enable-openal --enable-opencl --enable-opengl --enable-sdl2 --enable-libdc1394 --enable-libdrm --enable-libiec61883 --enable-nvenc --enable-chromaprint --enable-frei0r --enable-libx264 --enable-shared
  WARNING: library configuration mismatch
  avcodec     configuration: --prefix=/usr --extra-version=1ubuntu0.1 --toolchain=hardened --libdir=/usr/lib/x86_64-linux-gnu --incdir=/usr/include/x86_64-linux-gnu --arch=amd64 --enable-gpl --disable-stripping --enable-avresample --disable-filter=resample --enable-avisynth --enable-gnutls --enable-ladspa --enable-libaom --enable-libass --enable-libbluray --enable-libbs2b --enable-libcaca --enable-libcdio --enable-libcodec2 --enable-libflite --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-libgme --enable-libgsm --enable-libjack --enable-libmp3lame --enable-libmysofa --enable-libopenjpeg --enable-libopenmpt --enable-libopus --enable-libpulse --enable-librsvg --enable-librubberband --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libssh --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx265 --enable-libxml2 --enable-libxvid --enable-libzmq --enable-libzvbi --enable-lv2 --enable-omx --enable-openal --enable-opencl --enable-opengl --enable-sdl2 --enable-libdc1394 --enable-libdrm --enable-libiec61883 --enable-nvenc --enable-chromaprint --enable-frei0r --enable-libx264 --enable-shared --enable-version3 --disable-doc --disable-programs --enable-libaribb24 --enable-liblensfun --enable-libopencore_amrnb --enable-libopencore_amrwb --enable-libtesseract --enable-libvo_amrwbenc
  libavutil      56. 31.100 / 56. 31.100
  libavcodec     58. 54.100 / 58. 54.100
  libavformat    58. 29.100 / 58. 29.100
  libavdevice    58.  8.100 / 58.  8.100
  libavfilter     7. 57.100 /  7. 57.100
  libavresample   4.  0.  0 /  4.  0.  0
  libswscale      5.  5.100 /  5.  5.100
  libswresample   3.  5.100 /  3.  5.100
  libpostproc    55.  5.100 / 55.  5.100
Hyper fast Audio and Video encoder
usage: ffmpeg [options] [[infile options] -i infile]... {[outfile options] outfile}...

Use -h to get full help or, even better, run 'man ffmpeg'
```

I started using linux a month ago so still trying the sort it out. How can I find out what's wrong?

**Comment by axcore** [comment]
I have a fix for this. There will be a new release in a day or two.

**Comment by axcore** [comment]
Should be fixed in v2.3.085.

**Comment by harrwester** [comment]
Excellent, after couple of downloads it finally seems to work well. Thanks!
